logging/config/cluster-logging-systemd.adoc

```diff
@@ -1,12 +1,12 @@
 :_content-type: ASSEMBLY
```

Comment:
I can not find where this is referenced.  It looks like a hold over from 3.11 from before CRIO when you could configure nodes to write logs to journal.  If you can not find any reference I would consider removing it.